lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err, aclObject) {});
+ * myFile.acl.makePrivate(function(err, aclObject) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
+};
+
+/**
+ * Make a {module:storage/bucket} or {module:storage/file} object publicly
+ * readable.
+ *
+ * This is a short-hand method which will grant "reader" permissions to the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePublic
+ *
+ * @example
+ * myBucket.acl.makePublic(function(err, aclObject) {});
+ * myFile.acl.makePublic(function(err, aclObject) {});
+ */
+Acl.prototype.makePublic = function(callback) {
+  this.add({
+    scope: 'allUsers',
+    role: 'READER'
+  }, callback);
+};
+
+/**
  * Transform API responses to a consistent object format.
  *
  * @private
  */
 Acl.prototype.makeAclObject_ = function(accessControlObject) {
   var obj = {
-    scope: accessControlObject.scope,
+    scope: accessControlObject.entity,
```

Comment:
I believe we had regression tests checking the part of the response that was consistent, `role`. Our new tests check for `scope` as well.